lumicks/pylake/correlated_stack.py

```diff
@@ -193,6 +194,27 @@ def plot_tether(self, **kwargs):
         tether_kwargs = {"c": "w", "marker": "o", "mfc": "none", "ls": ":", **kwargs}
         plt.plot(x, y, **tether_kwargs)
 
+    def edit_image(self, frame=0, channel="rgb", show_title=True, **kwargs):
```

Comment:
My plan is to also include cropping in this widget so that you have kind of a one stop widget for all of your manipulation needs. I still don't necessarily like `edit_image` (it almost gives the feeling of you're doing inappropriate manipulation) but I can't think of anything better. Suggestions appreciated

Reply:
Yeah, this is a difficult one. `edit_image` just sounds very general to me. I hope we won't regret it later if we call it that.

I tried to come up with some names, but none of them make me go "yes, that's definitely it". Here's a random list of suggestions: `align_tether`, `level_tether`, `align_and_crop`, `crop_and_align`, `crop_and_level`, `reorient_tether`.